security-policies/bundle/compliance/policy/process/ensure_arguments_contain_key.rego

```diff
@@ -1,23 +1,23 @@
 package compliance.policy.process.ensure_arguments_contain_key
 
+import future.keywords.in
+
 import data.benchmark_data_adapter
 import data.compliance.lib.assert
 import data.compliance.lib.common as lib_common
 import data.compliance.policy.process.data_adapter
 
 process_args := benchmark_data_adapter.process_args
 
-finding(rule_evaluation) = result {
-	# set result
-	result := lib_common.generate_result_without_expected(
-		lib_common.calculate_result(rule_evaluation),
-		{"process_args": process_args},
-	)
-}
+finding(rule_evaluation) := lib_common.generate_result_without_expected(
+	lib_common.calculate_result(rule_evaluation),
+	{"process_args": process_args},
+)
 
 not_contains(entity) := assert.is_false(lib_common.contains_key(process_args, entity))
 
-contains(entity) := lib_common.contains_key(process_args, entity)
+# regal ignore:rule-shadows-builtin
```

Comment:
Should we remain instead?

Reply:
Rename? Honestly, I don't think that function adds much but abstraction. Should be replace by `object.keys` and `in`, but I did not want to make too many changes in this PR.

Reply:
My brain is a bit broken today and I copied that typo everywhere :laughing:.

Sure, let's leave it for now.